chore: new readme to better explain where to raise issues (#245)
botpress/documentation: readme.md

# Botpress Documentation

This repository contains the source code for the Botpress documentation website.

## Issues

Issues are disabled in the repository.

> If you find any problems in the documentation, have any suggestions about the documentation, have any questions about Botpress Cloud or want to report a bug in Botpress Cloud, please reach ou to us on [our discord server](https://discord.gg/botpress).

Reviewer: ```suggestion
If you find any problems in the documentation, have any suggestions for the documentation or Botpress Cloud in general, or want to report a bug, please reach out to us on [our discord server](https://discord.gg/botpress).
```

This is to ensure your issue is addressed as soon as possible and to keep all the information in one place.

## Contributions

> If you feel like adressing a documentation issue yourself, you can contribute by raising a [Pull Request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests) directly on this Github Repo.

Reviewer: ```suggestion
If you feel like addressing a documentation issue yourself, you can contribute by raising a [Pull Request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests) directly on this Github Repo.
```
